lua/ulx/modules/sh/freeze.lua

```diff
@@ -0,0 +1,28 @@
+CATEGORY_NAME = "CFC"
+
+local function freezeProps( callingPlayer, targetPlayers ) 
+    local entities = ents.FindByClass( "prop_*" )
+    local entCount = 0
+    local isTarget = {}
+    for _, ply in pairs( targetPlayers ) do
+        isTarget[ply] = true
+    end
+
+    for _, ent in pairs( entities ) do
+        if isTarget[ent:CPPIGetOwner()] then
+            local physicsObj = ent:GetPhysicsObject()
+            if IsValid( physicsObj ) then
+                physicsObj:EnableMotion( false )
+                physicsObj:Sleep()
+                entCount = entCount + 1
+            end
+        end
+    end
+
+    ulx.fancyLogAdmin( callingPlayer, "#A froze "..entCount.." props owned by #T", targetPlayers )
+end
+
+local entities = ulx.command( CATEGORY_NAME, "ulx freezeprops", freezeProps, "!freezeprops" )
```

Comment:
shouldn't this be named `freezeCmd`, `freeze`, or something similar instead of `entities`

Reply:
Yeah that would make some sense